modules/tinymce/src/plugins/table/test/ts/browser/ui/TableCellBackgroundColorTest.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+import { pAssertStyleCanBeToggledOnAndOffWithoutCheckmarks } from '../../module/test/TableModifiersTestUtils';
+
+describe('browser.tinymce.plugins.table.ui.TableCellBackgroundColorTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    toolbar: 'tablecellbackgroundcolor',
+    base_url: '/project/tinymce/js/tinymce',
+    menu: {
+      table: { title: 'Table', items: 'tablecellbackgroundcolor' },
+    },
+    menubar: 'table',
+    table_cell_background_color_map: [
+      {
+        title: 'Color',
+        value: '#ffffff',
+      }
```

Comment:
- [ ] Can we add a test with a color other than white as well as a test with an invalid colour (I believe it should default to white?)

Reply:
We'll test with a non-white property, to avoid the default. We won't test for an invalid color, the color-parsing is an external library.

Reply:
I don't really see the downside in adding a test for an invalid colour, I know it's an external library but we _do something_ with the result. Just another layer of protection basically for free

Reply:
I'll leave it up to you though